development/java/Rubberduck.Parsing/Grammar/build.gradle

```diff
@@ -0,0 +1,52 @@
+buildscript {
+    repositories {
+        mavenCentral();
+    }
+
+    dependencies {
+        classpath group: 'org.antlr', name: 'antlr4', version: '4.7.2'
+    }
+}
+
+plugins {
+    id 'java'
+}
+
+group 'com.rubberduckvba.rubberduck.parsing'
+version '1'
```

Comment:
should this possibly match to the build version of the main project? I'm thinking no, but I want to make sure everyone is aware that this is a thing we could try to enforce

Reply:
@Vogel612
As per antlr versioning, is there a easy to read place where the desired version number could be extracted from in the build script?

Reply:
You could try and use the `APPVEYOR_VERSION` environment variable that's available for AV builds and default to a "sensible default" for local builds

Reply:
@Vogel612 I have modified the build script to attempt this.